plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/DataTransferJob.java

```diff
@@ -106,12 +110,31 @@ private boolean transferData(DBRProgressMonitor monitor, DataTransferPipe transf
 
             IDataTransferProcessor processor = settings.getProcessor() == null ? null : settings.getProcessor().getInstance();
             try {
+                DBCStatistics totalStatistics = new DBCStatistics();
                 producer.transferData(
                     monitor,
                     consumer,
                     processor,
                     nodeSettings,
                     task);
+
+                totalStatistics.accumulate(producer.getStatistics());
+                totalStatistics.accumulate(consumer.getStatistics());
+
+                DBTTaskRun taskRun = task.getLastRun();
+                if (taskRun != null) {
+                    ArrayList<String> messageParts = new ArrayList<String>();
+                    if (totalStatistics.getRowsFetched() > 0) {
+                        messageParts.add(String.format("%s: %d", DTMessages.data_transfer_task_rows_fetched_message_part, totalStatistics.getRowsFetched()));
```

Comment:
The `String.format` is not NLS-friendly.

Consider using `NLS.bind` and provide a translation that contains a placeholder for a value: `Rows fetched: {0}`